src/commands/continue.yml

```diff
@@ -14,6 +14,13 @@ parameters:
     type: string
     description: "The domain of the CircleCI installation - defaults to circleci.com. (Only necessary for CircleCI Server users)"
     default: "circleci.com"
+  tag: 
```

Comment:
\~ I'd maybe be more specific here and call it `image_tag` or something like that, otherwise `tag` can be slightly ambiguous in this context.